### ISSUE TITLE
Overwrite dialogue dissapears as of 2.29.0

### ISSUE THREAD

**Issue by nambrosch** [filebrowser/filebrowser#3325]
In 2.29.0 and 2.30.0 the file overwrite dialogue does not stay on the screen.  It appears and vanishes before the user has a chance to interact with it.

Downgrading to 2.28.0 makes it stick again.

**Comment by nambrosch** [comment]
bump - issue still exists.

**Comment by siriusa51** [comment]
2.30.0 still exists

**Comment by nambrosch** [comment]
Yes, original post indicates issue exists in 2.29.0 and 2.30.0 but NOT 2.28.0.

**Comment by nambrosch** [comment]
@o1egl does the most recent release address this?

**Comment by tanst** [comment]
2.31.1 still exists

**Comment by nambrosch** [comment]
@o1egl has this been addressed?

**Comment by killua99** [comment]
bump, latest version has this issue 2.31.2

**Comment by nambrosch** [comment]
bump, assuming this is still an issue?

**Comment by killua99** [comment]
it's, the patch or PR does work but is not merged yet

**Comment by nambrosch** [comment]
Glad to hear it.  Was bumping this so it doesn't auto-close.
